cases/invalid-k8s-mangled-yaml/deployment.yaml

```diff
@@ -9,14 +9,29 @@ spec:
       app: nginx
   template:
     metadata:
-      lables:
+      labels:
         app: nginx
-    speck: 
-      conteiners:
+    spec:
+      containers:
       - name: nginx
-        image: nginx:1.14.2:invalid:tag
+        image: nginx:1.14.2
         ports:
         - containerPort: 80
+        env:
+        - name: API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: api-secrets
```

Comment:
these will fail because the secret doesn't exist too, add the secret alongside so that's not the discovered error